src/lib/select/option-group/option-group.ts

```diff
@@ -51,4 +53,18 @@ export class OptionGroupComponent extends BaseComponent implements IOptionGroupC
       this.setAttribute(OPTION_GROUP_CONSTANTS.attributes.LABEL, this._label);
     }
   }
+
+  /** The child options of this group. */
+  public declare options: ISelectOption[];
+
+  /** The builder function for the group content. */
+  public declare builder: ListDropdownOptionGroupBuilder;
+
+  /** The text content for the group. */
+  public get text(): string {
+    return this._label;
+  }
+  public set text(value: string) {
+    this.label = value;
+  }
```

Comment:
These properties can be set on the `<forge-option-group>` elements instead of forcing it to be done through JavaScript only.